service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/SocialWorkOtherControllerTest.java

```diff
@@ -0,0 +1,95 @@
+package uk.gov.hmcts.reform.fpl.controllers;
```

Comment:
could you write a test to see what happens when you have multiple documents uploaded with no titles